### PR TITLE
added background for cascade classifier

### PR DESCRIPTION
**Pull request by AnnaGsell**
please ckeck if it's to 'denglisch' ;)

### CODE REVIEW

File: chapters/background.tex [hunk 1]
```diff
@@ -1,4 +1,13 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
+Since Viola and Jones proposed "Rapid Object Detection using a Boosted Cascade of Simple Features" in 2001, their concept is the basis of most face detection approaches.
+It's an extremely fast and solid machine learning approach for object detection in videos and pictures, like e. g. faces.
```
Comment (mac641) [inline chapters/background.tex]: I'd suggest `[...], e.g. faces` or `[...], for example faces`

File: chapters/background.tex [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+
+In detail the concept is distinguished by three key components.
+The first is the “Integral Image”, a new image representation which allows the detector to compute features very fast.
+Secondly an AdaBoost based learning algorithm selects a small number of promising visual features and constructs very efficient classifiers.
+The last component is a method which combines increasingly more complex classifiers in a “cascade” where most of the computation time is spend on critical object-like parts of the given image by discarding non-promising regions early in the analysis process \cite{viola_jones}.
+
+For that reason the approach is highly appropriate to fulfill the face detection task of the soHappy application.
```
Comment (mac641) [inline chapters/background.tex:10]: I think this sentence sounds a little rough. You could use something like this: `Because of given reasons, this approach is recommended for most face detection purposes. Therefore we decided to use it as main algorithm in the soHappy application.`

**Review by mac641 — COMMENTED** [review]
I've seen some wording issues which could be improved.
But if you feel like this isn't required, I'm happy to approve this PR. Just let me know :)

File: sources/sources.bib [hunk 3]
```diff
@@ -25,3 +25,16 @@ @article{sample_ref
   publisher={Springer}
 }
 
+@inproceedings{viola_jones,
+  author = {Viola, Paul and Jones, Michael},
+  title = {Remember to Smile: Design of a Mobile Affective Technology to Help Promote Individual Happiness through Smiling},
+  year = {2017},
+}
+
+@inproceedings{opencv_cascade_classifier,
```
Comment (limkokyew) [inline sources/sources.bib:50]: When citing online sources, you usually have to specify when you retrieved it, e. g. "Retrieved September 05, 2020" due to websites being rather volatile. You can include this information by using:

```
@inproceedings{opencv_cascade_classifier,
  note = {Retrieved September 05, 2020}
}
```

File: chapters/background.tex [hunk 4]
```diff
@@ -1,4 +1,13 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
+Since Viola and Jones proposed "Rapid Object Detection using a Boosted Cascade of Simple Features" in 2001, their concept is the basis of most face detection approaches.
+It's an extremely fast and solid machine learning approach for object detection in videos and pictures, like e. g. faces.
+
+In detail the concept is distinguished by three key components.
```
Comment (limkokyew) [inline chapters/background.tex]: I'd suggest "In detail, the concept [...]".

File: chapters/background.tex [hunk 5]
```diff
@@ -1,4 +1,13 @@
...
+The first is the “Integral Image”, a new image representation which allows the detector to compute features very fast.
+Secondly an AdaBoost based learning algorithm selects a small number of promising visual features and constructs very efficient classifiers.
+The last component is a method which combines increasingly more complex classifiers in a “cascade” where most of the computation time is spend on critical object-like parts of the given image by discarding non-promising regions early in the analysis process \cite{viola_jones}.
```
Comment (limkokyew) [inline chapters/background.tex]: I think you can omit the "more" in "increasingly more".

"most of the computation time is **spent**"

File: chapters/background.tex [hunk 6]
```diff
@@ -1,4 +1,13 @@
...
+
+For that reason the approach is highly appropriate to fulfill the face detection task of the soHappy application.
+The open source computer vision and machine learning software library OpenCV provides a cascade classifier class for object detection, which uses the concept of viola and jones \cite{opencv_cascade_classifier}.
```
Comment (limkokyew) [inline chapters/background.tex]: Correct me if I'm wrong but I think that Computer Vision and Machine Learning are usually capitalized.

Viola and Jones have to be capitalized.

**Review by limkokyew — CHANGES_REQUESTED** [review]
Solid work! Just a couple of minor issues.

File: chapters/background.tex (hunk 5 above)
Comment (AnnaGsell) [reply chapters/background.tex]: Thank's. I'm sure 'increasingly more' is suitable here

File: chapters/background.tex (hunk 2 above)
Comment (AnnaGsell) [reply chapters/background.tex:10]: I get what you mean, but it's standard in paper (although I know it's not exactly a paper) to sound very convinced of the things you do and also never use 'we' etc. ...

File: sources/sources.bib [hunk 7]
```diff
@@ -25,3 +25,17 @@ @article{sample_ref
...
+  author = {OpenCV team},
+  title = {Cascade Classifier},
+  year = {2020},
+  url = {https://docs.opencv.org/3.4/db/d28/tutorial_cascade_classifier.html},
+  note = {Retrieved September 05, 2020},
```
Comment (limkokyew) [inline sources/sources.bib]: I listed September 05, 2020 as an example date - if that's when you looked at the website, then disregard this comment. Otherwise, update September 05, 2020 to the date where you looked at the website (presumably September 16, 2020 or something like that)

File: chapters/background.tex [hunk 8]
```diff
@@ -1,4 +1,13 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
+Since Viola and Jones proposed "Rapid Object Detection using a Boosted Cascade of Simple Features" in 2001, their concept is the basis of most face detection approaches.
```
Comment (limkokyew) [inline chapters/background.tex]: I think you could also write "Ever since Viola [...]". But this sentence is correct as well and it's up to your preferences whether you want to change it or not.

**Review by limkokyew — APPROVED** [review]
Looks good to me! Just two minor things left.

File: sources/sources.bib (hunk 7 above)
Comment (AnnaGsell) [reply sources/sources.bib]: you're absolutely right :D

File: chapters/background.tex (hunk 8 above)
Comment (craftamap) [inline chapters/background.tex]: Im pritty sure latex does not like quotes like this. Also, in line 7 and 9, you use different kind of quotation marks.
For english quotes in latex, please use ` `` ` for the first and `''` (two single quotes) as quotation marks. See here:
https://tex.stackexchange.com/questions/531/what-is-the-best-way-to-use-quotation-mark-glyphs

Comment (craftamap) [inline chapters/background.tex]: You could also try to use `\citetitle{viola_jones}` here instead of writing the title here. Let me know if it works.

**Review by craftamap — CHANGES_REQUESTED** [review]

File: chapters/background.tex (hunk 8 above)
Comment (AnnaGsell) [reply chapters/background.tex]: unfortunately not

**Review by diehmlukas — APPROVED** [review]

File: chapters/background.tex [hunk 9]
```diff
@@ -1,4 +1,13 @@
...
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
+Since Viola and Jones proposed ``Rapid Object Detection using a Boosted Cascade of Simple Features'' in 2001, their concept is the basis of most face detection approaches.
+It's an extremely fast and solid machine learning approach for object detection in videos and pictures, e. g. faces.
+
+In detail, the concept is distinguished by three key components.
+The first is the ``Integral Image'', a new image representation which allows the detector to compute features very fast.
+Secondly, an AdaBoost based learning algorithm selects a small number of promising visual features and constructs very efficient classifiers.
+The last component is a method which combines increasingly more complex classifiers in a ``cascade'' where most of the computation time is spent on critical object-like parts of the given image by discarding non-promising regions early in the analysis process \cite{viola_jones}.
```
Comment (mac641) [inline chapters/background.tex]: I'd use `increasingly` instead of `increasingly more`

**Review by craftamap — APPROVED** [review]

File: chapters/methodology.tex [hunk 10]
```diff
@@ -9,8 +9,20 @@ \subsection{Architecture}
 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 
 \subsection{State Machine}
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
```
Comment (mac641) [inline chapters/methodology.tex]: Methodology should be a separate PR

**Review by mac641 — CHANGES_REQUESTED** [review]

**Review by mac641 — APPROVED** [review]
👍🏿
